packages/shared/src/components/modals/ReportPostModal.tsx

```diff
@@ -43,67 +36,26 @@ export default function RepostPostModal({
     props.onRequestClose(event);
   };
 
-  // return (
-  //   <Modal
-  //     isOpen
-  //     kind={Modal.Kind.FlexibleCenter}
-  //     size={Modal.Size.Small}
-  //     onRequestClose={props.onRequestClose}
-  //     {...props}
-  //   >
-  //     <Modal.Header title="Report article" />
-  //     <Modal.Body>
-  //       <p className="mt-2 mb-6 text-theme-label-secondary typo-callout">
-  //         &quot;{post?.title}&quot;
-  //       </p>
-  //       <Radio
-  //         className="mt-2 mb-4"
-  //         name="report_reason"
-  //         options={reportReasons}
-  //         value={reason}
-  //         onChange={setReason}
-  //       />
-  //       <p className="px-2 mt-6 mb-1 font-bold typo-caption1">
-  //         Anything else you&apos;d like to add?
-  //       </p>
-  //       <textarea
-  //         onChange={(event) => setComment(event.target.value)}
-  //         className="self-stretch p-2 mb-4 w-full h-20 bg-theme-float rounded-10 resize-none typo-body"
-  //       />
-  //       <Checkbox
-  //         ref={inputRef}
-  //         name="blockSource"
-  //         className="self-center font-normal"
-  //       >
-  //         Don&apos;t show articles from {post?.source?.name}
-  //       </Checkbox>
-  //     </Modal.Body>
-  //     <Modal.Footer>
-  //       <Button
-  //         className="btn-primary"
-  //         disabled={!reason || (reason === 'OTHER' && !comment)}
-  //         onClick={onReportPost}
-  //       >
-  //         Submit report
-  //       </Button>
-  //     </Modal.Footer>
-  //   </Modal>
-  // );
-
   return (
-    <ReportModal {...props}>
-      <ModalCloseButton onClick={props.onRequestClose} />
-      <ReportHeading>Report article</ReportHeading>
-      <ReportDescription>&quot;{post?.title}&quot;</ReportDescription>
-      <div className="w-full">
+    <Modal
+      isOpen
+      kind={Modal.Kind.FlexibleCenter}
+      size={Modal.Size.Small}
+      onRequestClose={props.onRequestClose}
+      {...props}
+    >
+      <Modal.Header title="Report article" />
+      <Modal.Body>
+        <p className="mt-2 mb-6 text-theme-label-secondary typo-callout">
```

Comment:
I don't think we need the MT and spec has tertiary color.
```suggestion
        <p className="mb-6 text-theme-label-tertiary typo-callout">
```